code/game/machinery/modular_shield.dm

```diff
@@ -585,12 +598,26 @@
 		connected.shield_generator = null
 		connected.update_icon_state()
 
-/obj/machinery/modular_shield/module/charger
```

Comment:
why does it say i deleted this no i didnt , and when i check the file its still there :(